app/controllers/account/admin/issues_controller.rb

```diff
@@ -13,7 +14,8 @@ def edit; end
 
       def update
         if @issue.update_attributes(issues_params)
-          flash[:success] = 'Issue updated'
+          flash[:success] = 'Звернення змінено!'
+          facebook_posting(@issue) if @issue.status == 'open'
```

Comment:
Logic of changing status is pending - open - closed, so in theory issues should't have "open" status more than one time

But will try to check it. How better do it? Maybe add field to model "posted" - true/false?